### PR TITLE
crio: fix perf job

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

**Comment by sohankunkerkar** [comment]
/pj-rehearse pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes

**Comment by sohankunkerkar** [comment]
/pj-rehearse pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes

**Comment by sohankunkerkar** [comment]
@jtaleric, could you please take a look?
I believe this should pick up the CRI-O binary with the PR content we need.

File: ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml [hunk 1]
```diff
@@ -68,6 +68,24 @@ tests:
...
+      ENABLE_LOCAL_INDEX: "true"
+      ES_TYPE: qe
+      EXTRA_FLAGS: --churn-duration=20m --pod-ready-threshold=13s
+      ITERATION_MULTIPLIER_ENV: "15"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    - chain: openshift-qe-orion-consolidated
+    workflow: openshift-qe-installer-aws
+  timeout: 3h30m0s
```
Comment (jtaleric) [inline ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml:88]: Can you increase this to `8h` - the first attempt failed due to the `3h30m` https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/59005/rehearse-59005-pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes/1859057021169438720#1:build-log.txt%3A488

Regarding the versioning, can we confirm `cri-o://1.32.0-1.ci.el9` is the expected build? if so, lgtm!

Comment (sohankunkerkar) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml:88]: >Regarding the versioning, can we confirm cri-o://1.32.0-1.ci.el9 is the expected build?

Yup, that's expected. We recently merged a change that helps us identify the CRI-O binary built with the PR content.

Comment (sohankunkerkar) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml]: @jtaleric did you see the recent failure? https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/59005/rehearse-59005-pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes/1859242723970650112

It looks like CI couldn't find the `rhel-coreos-base` image.

**Comment by sohankunkerkar** [comment]
/pj-rehearse pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes

**Comment by sohankunkerkar** [comment]
/pj-rehearse pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes

File: ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml (hunk 1 above)
Comment (jtaleric) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml]: I am not sure why one run would succeed but following ones fail? Could be a build server thing?

Comment (jtaleric) [reply ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml]: It appears your new attempt got past that issue @sohankunkerkar

**Pull request merged**